### PR TITLE
Fix: Empty global values in CSS

### PR DESCRIPTION
**Pull request by kobizz**

### CODE REVIEW

File: core/files/css/base.php [hunk 1]
```diff
@@ -762,6 +763,11 @@ private function get_selector_global_value( $control, $global_key ) {
 
 		if ( ! empty( $control['groupType'] ) ) {
 			$property_name = str_replace( [ $control['groupPrefix'], '_tablet', '_mobile' ], '', $control['name'] );
+
+			if ( empty( $data['value'][ Global_Typography::TYPOGRAPHY_GROUP_PREFIX . $property_name ] ) ) {
```
Comment (matipojo) [inline core/files/css/base.php:767]: It doesn't cover `multiple` controls like `font-size`.

**Review by matipojo — CHANGES_REQUESTED** [review]

File: core/files/css/base.php (hunk 1 above)
Comment (kobizz) [reply core/files/css/base.php:767]: The data manager should return only changed values, without defaults

Comment (matipojo) [reply core/files/css/base.php:767]: It did it. If I understand correctly you only meant to check if `isset`?

Comment (matipojo) [reply core/files/css/base.php:767]:
```suggestion
			if ( ! isset( $data['value'][ Global_Typography::TYPOGRAPHY_GROUP_PREFIX . $property_name ] ) ) {
```

**Review by matipojo — APPROVED** [review]

**Pull request merged**